### PR TITLE
fix(app): typos

### PR DESCRIPTION
**Pull request by antleblanc**
💄 mostly typos and wordings :)

### CODE REVIEW

File: app/app.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ import {loginConfiguration} from './config/constants';
 let providers = [
   HTTP_PROVIDERS,
   provide(OvhRequestService, {
-    useFactory: (http) => {
+    useFactory: http => {
```
Comment (bnjjj) [inline app/app.ts]: why ?

Comment (bnjjj) [inline app/app.ts]: Alright after review all your changes, I want to keep this guideline. In fact I always want to put () ever if there are just 1 arg. I found that more clear. Could you let the () please ?

Comment (antleblanc) [inline app/app.ts]: done!
let me know if it's ok for you.
thx

**Comment by bnjjj** [comment]
Thank you !

**Pull request merged**